File: foodtruck/src/main/java/com/lucas/foodtruck/models/Lunch.java
package com.lucas.foodtruck.models;

import lombok.Data;

import javax.persistence.*;

import java.util.ArrayList;
import java.util.List;

@Entity
@Data
@Table(name = "LUNCH")
public class Lunch {
    @Id
    @GeneratedValue(strategy=GenerationType.IDENTITY)
    private Long id;
    private String name;

    @OneToMany(mappedBy="lunch")
    private List<Ingredient> ingredients = new ArrayList<>();

}

File: api/src/main/kotlin/com/foodtruck/api/models/Lunch.kt
package com.foodtruck.api.models

import lombok.Data
import org.springframework.data.cassandra.core.mapping.PrimaryKey
import org.springframework.data.cassandra.core.mapping.Table

@Data
@Table
class Lunch (
        @PrimaryKey
        private val id: Long,
        private val name: String,
        private val ingredients: List<Ingredient>
)